src/fromager/build_environment.py

```diff
@@ -173,7 +171,6 @@ def _createenv(self) -> None:
                 "--python",
                 sys.executable,
                 "--pip=bundle",
-                "--setuptools=bundle",
```

Comment:
This does not remove the bundled setuptools package with Python 3.11. You have to change the line to `--setuptools=none` to not install the bundled setuptools. Virtualenv has different defaults for Python 3.11 and 3.12, because Python 3.12's `venv` and `ensurepip` no longer install it either.

```
$ python3.11 -m virtualenv --help | grep -- --setuptools
...
  --setuptools version          version of setuptools to install as seed: embed, bundle, none or exact version (default: bundle)

$ python3.12 -m virtualenv --help | grep -- --setuptools
  --setuptools version          version of setuptools to install as seed: embed, bundle, none or exact version (default: none)
```

Reply:
Thanks, that's fixed